tests/testthat/test-namespace.R

```diff
@@ -122,7 +122,7 @@ test_that("unload() removes package environments from search", {
   pkgenv <- pkg_env("testNamespace")
   nsenv   <- ns_env("testNamespace")
   unload("testNamespace")
-  unload("compiler")
+  suppressWarnings(unload("compiler"))
```

Comment:
Not needed previously because `test-imports.R` did the same thing, and the warning only occurs on the first unload.